docs/ACCEPTANCE_TESTS.md

```diff
@@ -82,7 +82,7 @@ Sample outputs:
       ]
     },
     {
-      "noticeCode": "third_notice_code",
+      "noticeCode": "fourth_notice_code",
       "affectedSourcesCount": 3,
       "affectedSources": [
         {
```

Comment:
re: `acceptance-test-process.jpg` because I can't seem to add a comment directly to the JPEG. The flowchart is great! There is a typo for the word disappointing however: it should say `Is the code generating additional or disappearing errors?`

Reply:
Oops, my fault! No, it says disappearing but dissapearing